x/bank/legacy/v0_40/types.go

```diff
@@ -3,41 +3,6 @@ package v040
 // DONTCOVER
 // nolint
 
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
 const (
 	ModuleName = "bank"
 )
-
-var _ GenesisBalance = (*Balance)(nil)
```

Comment:
undo. The reason we copy the types is that they need to be pinned to their respective version, whereas if we simply reference the `types` package, they can change.